script_automation/esa_ubuntu_k3_single/_index.md

```diff
@@ -0,0 +1,45 @@
+# This example can be used to install ESA on an Ubuntu system with K3s. 
+
+This example requires an Ubuntu machine or VM running K3s. We will Arc Connect the system through this example. 
+You will need to create a storage account and provide your storage account name and container name in the below variables: 
+
+```bash
+export REGION="eastus"
+export RESOURCE_GROUP="myResourceGroup"
```

Comment:
shouldnt this be configurable also? My suggestion, crate a table for all the parameters explaining and providing an exmaple